src/ui/components/pages/otherAccounts.module.css

```diff
@@ -55,3 +62,9 @@
 .addAccountButton:hover {
   background-color: #edf0f4;
 }
+
+.addAccountButton.addAccountButtonSmall {
```

Comment:
Same thing here. Better split into `.addAccountButton`, `.addAccountButton_text` (or `*_full`, `*_normal`, whatever) and `.addAccountButton_icon` (or `*_small`) rules to avoid overriding. In general, adding is easier to reason about than overriding.